### PR TITLE
Replacing help & info icons with colored versions.

### PR DESCRIPTION
**Pull request by neoatomic**

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by neoatomic** [comment]
Did add some more icons to the page.

Examples. :

File: gui/slick/views/config.mako [hunk 1]
```diff
@@ -14,9 +14,9 @@
 % endif
 
 <div id="config-content">
-<table class="infoTable" cellspacing="1" border="0" cellpadding="0" width="100%">
+<table class="infoTable" cellspacing="1" border="0" cellpadding="0" width="145%">
```
Comment (OmgImAlexis) [inline gui/slick/views/config.mako:17]: This shouldn't be changed to more than `100%` instead the class should have the width changed or the parent div should have it's width changed.

Comment (miigotu) [inline gui/slick/views/config.mako:17]: Damn I missed that lol

**Comment by labrys** [comment]
Also the arguments line should wrap.

**Comment by OmgImAlexis** [comment]
@labrys shouldn't the argument line just use `<pre>` tags?

**Comment by miigotu** [comment]
That goes beyond the scope of this PR imho, the whole intention was just to add the different icons.

**Comment by MGaetan89** [comment]
Why not one sprite for all icons?
Maybe vertical center text with icons too? But that's debatable.

**Comment by neoatomic** [comment]
@OmgImAlexis How can i change the width correctly so the text stays on one line.? Could you post an example that i can test/play around with.?

**Comment by OmgImAlexis** [comment]
@neoatomic that main problem you'll have is screen width. It shouldn't matter if it's on one or multiple lines since depending on the amount of text and the screen width it could either be off the screen or be visible.

**Comment by labrys** [comment]
@OmgImAlexis Argument line would be fine to use `<pre>` tags, however then the css would have to handle the wrapping.  The argument line can potentially be really long, so wrapping should be enabled.  Also as @miigotu mentioned that's not really a part of this PR though, it was about the colored icons.

**Comment by OmgImAlexis** [comment]
@labrys just thought I'd mention it since the width got changed on here.

**Pull request merged**